src/routes/volume/detail/VolumeInfo.js

```diff
@@ -133,22 +133,49 @@ function VolumeInfo({ selectedVolume, snapshotModalState, engineImages, hosts, c
   }
 
   const volumeSizeEle = () => {
-    let oldSize = selectedVolume && selectedVolume.controllers && selectedVolume.controllers[0] && selectedVolume.controllers[0].size ? selectedVolume.controllers[0].size : ''
-    let isExpanding = (selectedVolume.size !== oldSize && selectedVolume.state === 'attached')
-    let message = `The volume is in expansion progress from size ${formatMib(oldSize)} to size ${formatMib(selectedVolume.size)}`
+    let expectedSize = selectedVolume.size
+    let currentSize = selectedVolume?.controllers[0]?.size ? selectedVolume.controllers[0].size : ''
```

Comment:
nit: You should be able to use optional and nullish operator, the same for other places. This can simplify the code. 

```js
let currentSize = selectedVolume?.controllers[0]?.size ?? ''
```